src/schema/imported/manifest.json

```diff
@@ -72,20 +54,31 @@
               "onError": "warn"
             },
             "version": {
-              "type": "string"
+              "type": "string",
+              "description": "Version string must be a string comprising one to four dot-separated integers (0-65535). E.g: 1.2.3.",
+              "format": "versionString"
             },
             "homepage_url": {
               "type": "string",
-              "format": "url",
-              "preprocess": "localize"
+              "format": null,
+              "preprocess": "localize",
+              "oneOf": [
+                {
+                  "format": "url"
+                },
+                {
+                  "pattern": "^__MSG_.*?__$"
```

Comment:
this pattern feels wrong to me but it's being imported right? Afair the string only has to start with ``__MSG_`` if translated and that doesn't have to be the case all the time.